main.go

```diff
@@ -191,6 +192,17 @@ func overrideSetOptions(flags []cli.Flag, cctx *cli.Context, cfg *config.Estuary
 				return fmt.Errorf("failed to parse announce address %s: %w", cctx.String("announce"), err)
 			}
 			cfg.Node.AnnounceAddrs = []string{cctx.String("announce")}
+		case "peering":
```

Comment:
`peering-peers` to match with ipfs config

Reply:
I followed the convention of the announce-addr. But I get your point of just using "peering-peers".